_pages/common-infrastructure-questions.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Common Infrastructure Questions we get from clients
+---
+
+### Do you support DNSSEC?
+
+DNSSEC compliance is currently impossible, and even if it was possible, it would be without merit on web based systems. If a client wants to set up DNSSEC systems in cloud.gov, for either internal or external based resolution, there's technically nothing that would prevent that. However, we strongly recommend not investing in it, as there are many things you should be doing instead to secure your communications. 
```

Comment:
or maybe a line at the top of the file saying "these apply to AWS and cloud.gov"?